src/themes/default.css

```diff
@@ -54,6 +54,16 @@
   --bl-size-5xl: 4rem; /* 64px */
   --bl-size-6xl: 5rem; /* 80px */
 
+  /* Z-indexes */
+  --bl-index-deep: -1;
+  --bl-index-default: 0;
```

Comment:
in the docs above, the value of default z-index is 1. we have to change that. However, I am thinking now that what is the use case of having this var? also, shouldn't the default value should be `auto`?

Reply:
Good catch and a valid question that I don't know the answer 😊 

Perhaps, for the time being, we should consider removing "deep" and "default" since I don't see any valid use cases for them. Furthermore, we do not currently use them in any of our components. Are you ok with that @buseselvi ?

Reply:
Normally we are inspired by [Nord Health Design System](https://nordhealth.design/tokens/#z-index) here but the explanation on their document is also not very convincing to me.

Reply:
technically the only use-case I can think of is that you want to take the element out of its natural stack order and put it in the base, so maybe we can still keep it but change the name to `--bl-index-base` while considering our base value as 0, wdyt?

Reply:
because the name `-default` can be confusing here and ppl start using it even when it's not necessary.

Reply:
Yeah, makes sense to me. I pushed some changes. Please check the current version. @buseselvi @leventozen